zanata-war/src/main/java/org/zanata/events/webhook/DocumentMilestoneEvent.java

```diff
@@ -77,4 +78,8 @@
      */
     private String editorDocumentUrl;
 
+    @Override
+    public String getType() {
+        return this.getClass().getName();
```

Comment:
This 'type' property is for JSON, is it? If so, I think we should have a constant, rather than depending on the class name. eg, if we refactor the package names, our JSON serialisation will change.  What about a simple constant "DocumentMilestoneEvent"? Would that do?